modules/apps/transfer/keeper/relay.go

```diff
@@ -158,6 +158,84 @@ func (k Keeper) sendTransfer(
 	return sequence, nil
 }
 
+// OnSendPacket handles transfer sending logic.
+// TODO: Copy godoc from above sendTransfer function.
+func (k Keeper) OnSendPacket(ctx sdk.Context,
+	sourcePort string,
+	sourceChannel string,
+	packetData types.FungibleTokenPacketData,
+	sender sdk.AccAddress,
+) error {
```

Comment:
we could change function sig to just accept same args as `sendTransfer` does now and handle getting those out of the packet data in the `IBCModule` entrypoint